projects/sample/ReadMe_SampleProject.md

```diff
@@ -38,18 +43,24 @@ There are six main components to the sample project, which are delineated below.
             - In this example, you will need to use the full path to the `sample` directory.
         - *allowed file extensions* → The list of extensions for the file types that will be allowed. Again for security, the fileserver will otherwise restrict which file types it will return.
             - In this example, you will need to allow the following extensions: `.dcm`, `.txt`, and `.mat`
+        - IMPORTANT: The previous notes are important because the projectInterface, which runs from the cloud, *needs* to get permission about what can be read or written to the console computer.
 
 3. When you navigate your web browser to the URL [http://localhost:8888/](http://localhost:8888/), you will see the following page:
 ![](ReadMe_Images/rtCloudWeb_login.png)
 Log in using the username and password you set up during the initial installation.
 
-4. On the web browser, click on **Initialize** in the `Session` tab to run `initialize` python script. 
+4. On the web browser, go to the `Settings` tab to look at the default values that were read from the configuration `toml` file on the cloud.
+![](ReadMe_Images/rtCloudWeb_settings.png)
+
+- For this sample project, you don't have to change any of the settings. However, this is the location where you can load a new configuration file by using **Choose File** or reset the default settings after making changes on this page by pressing **reloadDefaultConfig**.
```

Comment:
Maybe:
"However, this is the location where you can edit configurations (there is no 'save' button, they will take effect on your next run). You can also load a new configuration file by using **Choose File** or reset to the original default settings by pressing **reloadDefaultConfig**.